shared/src/main/scala/squants/market/MoneyContext.scala

```diff
@@ -34,6 +35,26 @@ case class MoneyContext(
     allowIndirectConversions: Boolean = true) {
 
   /**
+    * Custom implementation using SortedSets to ensure consistent output
+    * @return String representation of this instance
+    */
+  override def toString: String = string
```

Comment:
Yes, much cleaner.  I needed to through a toList call in the pipeline to provide sortability without the Ordering implicit.  I'll have a new push soon.